skyvern/forge/sdk/workflow/models/block.py

```
@@ -3049,6 +3050,223 @@ async def execute(
+                self.format_block_parameter_template_from_workflow_run_context(recipient, workflow_run_context)
+            )
+
+        self.recipients = formatted
+
+    async def execute(
+        self,
+        workflow_run_id: str,
+        workflow_run_block_id: str,
+        organization_id: str | None = None,
+        browser_session_id: str | None = None,
+        **kwargs: dict,
```

Ruff (0.14.1)

Unused method argument: `kwargs`

(ARG002)